tests/core/test_utils.py

```diff
@@ -2,6 +2,7 @@
 from __future__ import unicode_literals
 import unittest
 import os
+import pytest
```

Comment:
Just a small tip: you shouldn't include pytest here.

You can run tests using:
- ``pytest tests/path/to/my/test`` under the virtualenv to run a single test. 
- ``pytest tests/`` under the virtualenv to run all the tests.
- ``make tests`` to run all the tests for all python versions, like travis do.

Hope this helps! :)